planning/behavior_path_planner/include/behavior_path_planner/parameters.hpp

```diff
@@ -26,8 +26,12 @@ struct BehaviorPathPlannerParameters
   double minimum_pull_over_length;
```

Comment:
@yukkysaito 
What do you think about this PR beging merged to .universe? (map coordinate drivable area and dynamic size of drivable area)

Reply:
Great! Very good change.